### ISSUE TITLE
jwrosewell: Do not reference S&P questionnaire

### ISSUE THREAD

**Issue by ekr** [patcg/patwg-charter#10]
Note: I am transcribing comments from the gdoc to issues.

@jwrosewell writes:
> For W3C see FOs related to other working group charters. [https://www.w3.org/Member/wiki/DirectorFOdashboard](https://www.google.com/url?q=https://www.w3.org/Member/wiki/DirectorFOdashboard&sa=D&source=docs&ust=1647900053853167&usg=AOvVaw3mqZ47RoLnwsA10IcWh2Sw) - Until these are settled the co-ordination should remove reference to the S&P questionnaire and other documents that take an anti competitive position.

**Comment by darobin** [comment]
One thing that should be clarified here is that even if those groups were to stop mentioning the Security & Privacy questionnaire in their charters, they would _still_ be under Wide Review obligations that include… the Security & Privacy questionnaire. It's mentioned for clarity, but it's required by the TAG anyway.

Another thing that is worth mentioning is that in every case we have one single member/individual pushing the same objection over and over again and entire groups comprised of dozens of members and people who have consensus to keep the questionnaire. So the obvious conclusion to draw from the FOs is that the questionnaire is very popular in the Membership (and for good reason — it codifies important knowledge built by a broad community from many different backgrounds).

**Comment by jwrosewell** [comment]
The Formal Objection (FO) referenced relates to the continued use of the S&P questionnaire across the W3C including by this group. The preferred method of resolution is for the S&P questionnaire to be rewritten to avoid steering proposers towards solutions that restrict data sharing which in turn interferes with competition. The "broad community" @darobin refers to does not include competition lawyers. This is particularly relevant to this group which seeks to justify proposals that restrict competition and innovation on the basis of privacy. It is particularly important when members of the group seek to address perceived market failures in the provision of digital advertising under the guise of privacy.

Unfortunately we do not have a mechanism for anonymous Formal Objection. We also have an environment of fear and intimidation where long time W3Cers seek to impose their views on the rest of the group and silence them when they can not provide evidence to justify their long held beliefs that have been codified into the W3C environment. We have an environment where competition is rarely discussed openly. I therefore don't believe that the number of voices in support of this issue should be considered the only method to assess it.

However as the matter is with the W3C Director for resolution the group should wait for their decision. I favor the W3C Director setting up a Legal Advisory group to advise the authors of the S&P questionnaire and other documents on matters related to law. This group would then steer these documents towards a position that preserves competition and aligns the W3C to privacy laws.

**Comment by AramZS** [comment]
As a not-long-time W3Cer I will note I feel no fear or intimidation or any concern with continued use of the S&P questionnaire. I have no interest in silencing you, however, as @darobin notes, the S&P enjoys a broad consensus and is a useful document with important knowledge across the many backgrounds and participants in the W3C. 

That said, it appears that the existing process for this under the W3C is for the group to adopt the use of the S&P and for your formal objection to be raised to the Director. Until such a decision is rendered by the W3C Director, one way or another, I see no reason to not continue to allow that to be the process. I see only @jwrosewell / 51Degrees' objection to use of the S&P and I think we can note here that this document will be altered--if needed--by any decision by the W3C but should go forward as is standard until such a time as that decision occurs. I assume that any decision would impact all groups leveraging the S&P if it were to occur and we would make whatever changes the other groups would make at that time. However, @jwrosewell you can raise another objection on this matter to the level of the other complaints raised. 

I will note on the record that your objection here is raised, however, I do not wish to bring up this topic when we discuss the WG charter on the upcoming call, as you've noted @jwrosewell this is well-tread ground and at the level of the W3C Director, so we will not be able to make a decision attempting to anticipate the W3C Director's actions but instead move forward using the current state of reality. I do not anticipate that taking up limited time on the upcoming call will move this forward in any more meaningful way then discussions in other groups and so I do not think it is worth discussing there.

**Comment by AramZS** [comment]
> However as the matter is with the W3C Director for resolution the group should wait for their decision. I favor the W3C Director setting up a Legal Advisory group to advise the authors of the S&P questionnaire and other documents on matters related to law. This group would then steer these documents towards a position that preserves competition and aligns the W3C to privacy laws.

This would stop almost all work in the W3C pending such a decision. We're not seeing that happening and have not been instructed to do so. As such, we aren't going to be able to engage with this question. This is the wrong level for this concern.

**Comment by jwrosewell** [comment]
I consider it a shame that the W3C Antitrust Guidelines, which are short and easy to understand, are rarely mentioned at the start of meetings or action taken under them, and that the S&P questionnaire continues to be so widely referenced. I sincerely hope the W3C will adopt practices and guidelines akin to a modern technology standards setting organisation for a resource used by 5bn+ people in the not too distant future.

How to handle this issue is a matter for the chairs. I would prefer it to remain open pending the resolution of the FO by the W3C Director. I'm not sure there's much more to debate at this time.

**Comment by AramZS** [comment]
I think it is assumed that all members review the Antitrust guidelines as part of their sign up process, it seems duplicative to mention them again at the top of every meeting. I understand your objection but it sounds like we agree that this is not useful as an issue for further conversation at this time so I'm not going to open it up to discussion in the upcoming real-time meeting. I'm going to mark it as blocked and I will leave it open pending a resolution of the FO by the W3C Director.

**Comment by jwrosewell** [comment]
Purely FYI. Other organisations like 3GPP do make such a statement.

https://www.3gpp.org/about-3gpp/legal-matters/21-3gpp-calendar/1616-statement-of-antitrust-compliance

I consider the W3C an exception in this regard. There is an advantage to being reminded repeatably of dangers as anyone who has travelled on the London Underground can attest. Mind the gap.

**Comment by AramZS** [comment]
We have added the antitrust rules into the charter as an additional mention, we do not reference the S&P questionnaire as far as I can see beyond where it may be referenced in other documents we refer to outside of my knowledge. This, as @martinthomson has stated, appears to be in consideration of a larger W3C process beyond the scope of our capacity to discuss or alter. As such, I believe this thread has been resolved.
